httpsproxy/caddyfile

```diff
@@ -0,0 +1,4 @@
+mysite.example.com {
+    
+    proxy / mysite.example.com:3333
+}
```

Comment:
I wasnt sure if that syntax could be used for more than 1 site, as I have multiple sites (each with {} ) in my own caddy file.  

I will update the file in the example though.